app/main/views/services.py

```diff
@@ -724,31 +811,11 @@ def set_remove_last_subsection_error_message():
             else:
                 abort(e.status_code)
 
-    else:
-        section_responses = [field for field in containing_section.get_field_names() if field in draft]
-        fields_remaining_after_removal = [field for field in section_responses if field not in fields_to_remove]
-
-        if draft['status'] == 'not-submitted' or len(fields_remaining_after_removal) > 0:
-            # Show page with "Are you sure?" message and button
-            return redirect(
-                url_for('.view_service_submission',
-                        framework_slug=framework_slug,
-                        lot_slug=lot_slug,
-                        service_id=service_id,
-                        confirm_remove=question_slug,
-                        section_id=section_id
-                        )
-            )
-        else:
-            # You can't remove the last one
-            set_remove_last_subsection_error_message()
```

Comment:
It looks as if there's only one use of `set_remove_last_subsection_error_message()` left. Does it still need to exist?